src/test/java/TestAhoCorasickDoubleArrayTrie.java

```diff
@@ -101,6 +102,51 @@ public void hit(int begin, int end, String value)
         });
     }
 
+    private static class CountHits implements AhoCorasickDoubleArrayTrie.IHitCancellable<String> {
+        private int count = 0;
+        private boolean countAll = true;
```

Comment:
attributes should be initialized in the constructor, not here.

Reply:
Good point